src/services/navigator/helpers.test.ts

```diff
@@ -863,3 +873,55 @@ describe('mergeDocuments', () => {
     });
   });
 });
+
+describe('add route', () => {
```

Comment:
minor/nit: the first arg of the top level `describe` is generally the function name that's being tested
 ```suggestion
describe('addStackRoute', () => {
```

Reply:
Fixed